### PR TITLE
Ltd 4839 duplicated new flow

### PR DESCRIPTION
**Pull request by Tllew**
### Aim

[LTD-](https://uktrade.atlassian.net/browse/LTD-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,12 +253,48 @@ ${ADDITIONAL_PYTEST_UI_TEST_ARGS} is always set to --headless in the docker-comp
...
+```
+java -jar /Users/lngo_dit/Downloads/selenium-server-4.19.1.jar standalone --selenium-manager true
+```
+
+change in the code to you local server for example:
+
+```
+command_executor="http://192.168.1.41:4444"
+```
+
```
Comment (hnryjmes) [inline README.md:297]: this seems pretty unexpected to me, I'm not sure java is needed to do this. there is an existing way to run UI tests on Selenium with browser showing by using VNC

Comment (Tllew) [reply README.md:297]: looks like java is required for selenium grid
https://www.selenium.dev/selenium/docs/api/py/index.html#selenium-grid-optional

this was some of the additional work that was carried over when I used @seijihg's branch.

it does seem required to run grid locally.

Comment (hnryjmes) [reply README.md:297]: ok, I still don't see how this is relevant to this ticket. but if you think it's needed then we can keep it in.

**Review by hnryjmes — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (seijihg) [reply README.md:297]: @hnryjmes haha prob should have added this in the seperate Ticket :P was too lazy xD its jut a readme update anw.

**Pull request merged**